providers/ses/src/lib/ses.provider.ts

```diff
@@ -59,10 +59,35 @@ export class SESEmailProvider implements IEmailProvider {
   async checkIntegration(
     options: IEmailOptions
   ): Promise<ICheckIntegrationResponse> {
+    let success: boolean;
+    let message: string;
+    let code: CheckIntegrationResponseEnum;
+    const transporter = nodemailer.createTransport({
+      SES: { ses: this.ses, aws: { SendRawEmailCommand } },
+    });
+
+    const testResponse = await transporter.sendMail({
+      html: '',
+      text: 'This is an Email to test the integration of your Amazon SES',
+      to: this.config.from,
+      from: this.config.from,
+      subject: 'Test Amazon SES integration',
```

Comment:
you could use the `options` parameter of the function to get these values instead of manually writing it here.
e.g,
```js
to: options.to,
```

if not then remove the `options` parameter.